lib/Conversion/SCFToCalyx/SCFToCalyx.cpp

```diff
@@ -1091,6 +1091,11 @@ LogicalResult BuildOpGroups::buildOp(PatternRewriter &rewriter,
       /*out=*/remPipe.out_remainder());
 }
 
+// https://github.com/llvm/circt/issues/2660
```

Comment:
```suggestion
// Returns the bit width for the given dimension. This will always be greater than zero. 
// See: https://github.com/llvm/circt/issues/2660

```